.gitignore

```diff
@@ -1,6 +1,7 @@
 *.class
 *.log
-project/
+project/*
+!project/plugins.sbt
```

Comment:
I'd also add [project/build.properties](https://www.scala-sbt.org/1.x/docs/Basic-Def.html#Specifying+the+sbt+version)